build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -92,20 +92,21 @@ Future<void> _createDevCompilerModule(Module module, BuildStep buildStep,
     {bool debugMode = true}) async {
   var transitiveDeps = await buildStep.trackStage('CollectTransitiveDeps',
       () => module.computeTransitiveDependencies(buildStep));
-  var transitiveKernelDeps = transitiveDeps.map(
-      (module) => module.primarySource.changeExtension(ddcKernelExtension));
+  var transitiveKernelDeps = [
+    for (var dep in transitiveDeps)
+      dep.primarySource.changeExtension(ddcKernelExtension)
+  ];
   var scratchSpace = await buildStep.fetchResource(scratchSpaceResource);
 
-  var allAssetIds = Set<AssetId>()
-    ..addAll(module.sources)
-    ..addAll(transitiveKernelDeps);
+  var allAssetIds = {...module.sources, ...transitiveKernelDeps};
```

Comment:
consider an explicit `<AssetId>` type or something here? At a glance its not clear if this is a set or a map.

Reply:
Sure.